edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/ConfigUpdater.cs

```diff
@@ -56,6 +56,13 @@ public async Task Init(IConfigSource configProvider)
                     {
                         Events.GettingConfig();
                         await pullTask;
+
+                        this.currentConfig.Expect<InvalidOperationException>(() => throw new InvalidOperationException(
```

Comment:
Suggested text - Cannot start successfully as could not get configuration for EdgeHub.

Reply:
This is the same text that is used in later versions. Should we change those also?